### PR TITLE
Landing page for v1 (#30)

### PR DESCRIPTION
**Pull request by vinaypuppal**
From @M-ZubairAhmed 
* (WIP) front page dev

* 1. Changes nav bar links from all caps to title case
2. nav links reordered to home,learn,space,events,blog
3. blog opens in same page

* 1. removed learn stock pics directory
2. added remaining props to row-events.js

### CODE REVIEW

**Comment by vinaypuppal** [comment]
@M-ZubairAhmed and @duttakapil merged #30  and reopened here to merge to master because of reason specified here https://github.com/coderplex/coderplex/pull/29#issuecomment-338057219

Push any changes here

File: components/header.js [hunk 1]
```diff
@@ -46,9 +41,14 @@ export default props => {
...
     {
+      title: 'Events',
+      path: '/events',
+      external: false,
+    },
+    {
       title: 'Blog',
       path: 'https://medium.com/coderplex',
-      external: true,
+      external: false,
```
Comment (vinaypuppal) [inline components/header.js]: Why is this changed to `false`?

Comment (M-ZubairAhmed) [reply components/header.js]: opening blog feels its taking away from web site, i feel it should be continuous
